addons/mass_mailing/static/tests/mass_mailing_favourite_filter_tests.js

```diff
@@ -407,5 +407,72 @@ QUnit.module('favorite filter widget', (hooks) => {
         assert.isNotVisible(fixture.querySelector('.o_mass_mailing_remove_filter'),
             "should not have option to remove filter because mailing domain is changed");
     });
+
+    QUnit.test('mailing_model_name can be empty without throwing an error', async (assert) => {
+        assert.expect(1);
```

Comment:
Because of other suggestions
```suggestion
        assert.expect(4);
```